src/events/request/RequestEventHandler.ts

```diff
@@ -58,6 +60,32 @@ export default class RequestEventHandler implements EventHandler<'message'> {
 			return;
 		}
 
+		if ( BotConfig.request.invalidRequestJql ) {
+			const tickets = Array.from( this.getTickets( this.replaceTicketReferencesWithRichLinks( origin.content, regex ) ) );
+			const searchResults = await this.jira.search.search ( {
+				jql: `(${ BotConfig.request.invalidRequestJql }) AND key in (${ tickets.join( ',' ) })`,
+				fields: ['key'],
+			} );
+			const invalidTickets = searchResults.issues.map( ( { key } ) => key );
+			if ( invalidTickets.length > 0 ) {
+				try {
+					await origin.react( BotConfig.request.noLinkEmoji );
+				} catch ( error ) {
+					this.logger.error( error );
+				}
+
+				try {
+					const warning = await origin.channel.send( `${ origin.author }, your request (<${ origin.url }>) contains a ticket that is less than 24 hours old. Please wait until it is at least one day old before making a request.` );
+
+					const timeout = BotConfig.request.noLinkWarningLifetime;
```

Comment:
As we're using `noLinkWarningLifetime` here too, let's rename it to something like `requestWarningLifetime` :eyes:

Reply:
K, I'll change the emoji's name as well